api/src/.vscode/settings.json

```diff
@@ -1,5 +1,5 @@
 {
-    "python.pythonPath": "env\\Scripts\\python.exe",
+    "python.pythonPath": "d:\\umigratecode\\umigrate\\api\\src\\venv\\Scripts\\python.exe",
```

Comment:
Please undo this change. Thanks!

Reply:
You might have missed adding back `"python.pythonPath": "env\\Scripts\\python.exe"`